scripts/codestyle/run_uncrustify.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+if [ $# -gt 3 ]; then
+	echo "Too many parameters specified"
+	exit 1
+fi
+
+VERBOSE=false
+HASHES=()
+
+while [[ $# -gt 0 ]]
+do
+	key="$1"
+	case $key in
+	    -v|--verbose)
+	    VERBOSE=true
+	    ;;
+	    *)
+	    HASHES+=($key)
+	    ;;
+	esac
+	shift
+done
+
+if [ ${#HASHES[@]} -eq 2 ];
```

Comment:
Thanks!
Could you change help format to more formal one? Like (`man git-diff`):
``` 
      git diff [options] [<commit>] [--] [<path>...]
      git diff [options] [--no-index] [--] <path> <path>
```
where optional parts are in square brackets? I suppose
```
     $0 [<SHA1> <SHA2>]
``` 
should work?